internal/nri/domain.go

```diff
@@ -0,0 +1,109 @@
+package nri
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/cri-o/cri-o/internal/log"
+	"github.com/sirupsen/logrus"
+
+	nri "github.com/containerd/nri/pkg/adaptation"
+)
+
+// Domain implements the functions the generic NRI interface needs to
+// deal with pods and containers from a particular containerd namespace.
```

Comment:
comment needs updating. We don't have containerd namespaces :)

Reply:
Ouch... thanks! Updated comments.